index.html

```diff
@@ -25,54 +31,44 @@
             <p>Ayudando a los Hispanohablantes a Aprender Terminolog&iacute;a de Desarollo Web</p>
         </div>
     </header>
-    <div class="nav-placeholder" style="height: 42px;">
+<h4 class='searchlabel'>Select an element to view definition, memorization aids, audio pronunciation, and code sample:<br>Selecione un elemento para ver la definici&oacute;n, ayuda para recordar, la pronunciaci&oacute;n con audio, y ejemplos de como usar la etiqueta:</h4>
+
+<div class="sticky-combo">
         <nav class="">
             <div class="nav-inner">
                 <div class="nav-inner-most">
```

Comment:
@ahaynes-gpsw so this is the link to the tut I followed. https://www.youtube.com/watch?v=utonytGKodc
when I remove a div or 2 I get the same problems I had before when the nav would jump out of place as well as the body. This is the best way I was able to solve that problem :) I can mos def check out something that would not need so much nesting. (im still pretty new to this, still learning)